source/installation/install-software.rst

```diff
@@ -91,55 +92,51 @@ software requirements:
 
             sudo apt install ondemand
 
-.. note::
-
-   If *authenticating against LDAP* or *wishing to evaluate OnDemand* using `ood` user, you **must install** `ondemand-dex` as 
-   described below. See :ref:`add-ldap` for details on configuration of LDAP and :ref:`start-services` for the `ood` user.
-
-#. (Optional) Install :ref:`authentication-dex` package
+3. Start services
+-----------------
 
-   **RedHat/CentOS/Rocky only**
+   .. tabs::
 
-    .. code-block:: sh
+      .. tab:: RHEL/CentOS 7
 
-       sudo yum install ondemand-dex
+        .. code-block:: sh
 
-   **Ubuntu only**
+          sudo systemctl start httpd24-httpd
+          sudo systemctl enable httpd24-httpd
 
-    .. code-block:: sh
 
-       sudo apt install ondemand-dex
+      .. tab:: RHEL/CentOS/Rocky 8
 
-#. (Optional) Install OnDemand SELinux support if you have SELinux enabled. For details see :ref:`ood_selinux`
+         .. code-block:: sh
 
-   **RedHat/CentOS/Rocky only**
+          sudo systemctl start httpd
+          sudo systemctl enable httpd
 
-    .. code-block:: sh
+      .. tab:: Ubuntu
 
-       sudo yum install ondemand-selinux
+         .. code-block:: sh
 
-.. warning::
+          sudo systemctl start apache2
+          sudo systemctl enable apache2
 
-   For some older systems, user ids (UID) may start at ``500`` and not the
-   expected ``1000``. If this true for your system, you will need to modify the
-   :file:`/etc/ood/config/nginx_stage.yml` configuration file to allow these
-   users access to OnDemand:
+4. Verify installation
+----------------------
 
-   .. code-block:: yaml
-      :emphasize-lines: 9
+Now that Open OnDemand is installed and apache running, it should be serving
+a public page telling you to come back here and setup authentication.  If this
+is the case - then you should :ref:`secure your apache <add-ssl>` then :ref:`add authentication <authentication>`.
 
-      # /etc/ood/config/nginx_stage.yml
-      ---
+If you're seeing the default apache page (Ubuntu users will) you will have to :ref:`debug virtualhosts <show-virtualhosts>`
+and likely :ref:`configure a servername <ood-portal-generator-servername>`.
 
-      # ...
 
-      # Minimum user id required to generate per-user NGINX server as the requested
-      # user (default: 1000)
-      #
-      min_uid: 500
+.. warning::
 
-      # ...
+    For some older systems, user ids (UID) may start at ``500`` and not the
```

Comment:
I wonder if we can remove this entire paragraph?  The last OS that started at 500 was RHEL6 and we long dropped RHEL6 support. RHEL7+ start at 1000.

Reply:
yep